### ISSUE TITLE
GD-214: Test Root Folder text field does not display the right most character of the entered value

### ISSUE THREAD

**Issue by Zovix** [MikeSchulze/gdUnit3#214]
**The used Godot version:**
3.4.2.stable

**OS including version:**

Windows 11 - 21H2
Resolution: 2560x1440
Scale: 110%

**Describe the bug**
In the GdUnit3 settings, the Test Root Folder text field does not display the right most character of the entered value after re-entering the settings.

**Steps to Reproduce**
1. Go to GdUnit3 -> Settings -> Common
2. See error in image. "test" is the full string that was entered in the text area.

![image](https://user-images.githubusercontent.com/590152/150667949-22f68ca3-4d6a-4ed4-ada3-9c0ad32d3442.png)

**Comment by MikeSchulze** [comment]
Thanks for reporting this issue,

I'm not able to reproduce.
Can you please provide the screen resolution you use?

**Comment by Zovix** [comment]
Here are my current settings.

Godot:
![image](https://user-images.githubusercontent.com/590152/150683674-b1432ae9-b915-4e9e-97d7-8b67149eb8fe.png)


Windows:
Resolution: 2560x1440
Scale: 110%

**Comment by MikeSchulze** [comment]
You wrote a scale of 110% but the screenshot shows 100%?

**Comment by Zovix** [comment]
The 110% scaling is from the Windows 11 Display Settings scaling.

I actually found that this was caused due to the custom "Main Font" that I was using in Godot. If I set it to the default font then it is displaying correctly.

Not expecting support for it but if you're interested:
[Caskaydia Cove Nerd Font Complete Mono.zip](https://github.com/MikeSchulze/gdUnit3/files/7920817/Caskaydia.Cove.Nerd.Font.Complete.Mono.zip)
